lib/Common/Exceptions/ReportError.cpp

```diff
@@ -19,6 +19,9 @@ void ReportFatalException(
     if (IsDebuggerPresent())
     {
         DebugBreak();
+#if DBG
+        return;
+#endif
```

Comment:
that's just easier to debug, forgot to remove this after rebase/merge with my local change